sources/common/src/EventSignalCreator.cpp

```diff
@@ -28,10 +28,12 @@
 #include "EventSignal.h"
 
 SIGNAL::Signal *PARSER::EventSignalCreator::Create(const std::string &rLogLine,
+                                                   INSTRUMENT::Instrument::LineNumberT lineNumber,
                                                    SIGNAL::SourceRegistry::HandleT sourceHandle) const
 {
     std::smatch result;
 
+    (void)lineNumber;
```

Comment:
It is the implementation of the pure virtual interface defined in **SignalCreator.h**. For normal signals created on the data provided by the standard vcdMaker (as in this case) format it is not needed.

However it can be used in the user defined log format to create the signal. Let's imagine the log not containing timestamps (but just names and values):
```
Signal.A 10
Signal.A 20
Signal.A 30
...
```

A user can describe the timestamp e.g. in the following way: `50 * line() + 25`.
It will create timestamps based on the log line number. So, in the case above the first signal will be logged at the time of 25 [time units]. The second one at 75 [time units]. And so on...